dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/AuditReplayReducer.java

```diff
@@ -9,6 +9,12 @@
 
 import java.io.IOException;
 
+
+/**
+ * <p>AuditReplayReducer aggregates the returned latency values from {@link AuditReplayMapper} and sums
```

Comment:
No need to start with a `<p>`. Only needed to make a subsequent paragraph break.